src/main/resources/endpoint.properties

```diff
@@ -0,0 +1,18 @@
+# System properties
+_endpoint_name=veritone
+_app_name=test1
+_environment=dev
+_pod_id=id
+_profile=default
+_custom_domain=
+
+_debug=true
+_local_deployment=true
+
+_webservices_port=10000
+_base_domain=localhost:8000
+_endpoints_services_api=http://localhost:2233/api
+_token=test1/dev/veritone
+
+# Endpoint specific properties
+_endpoint_config={"environment":"stage", "region":"us-1", "apiToken":"9kWCQoSyXrlzFfgPrE3VR80DYwr4sdhPWcOt7kfpFsxUvmbL3c-jJw"}
```

Comment:
I would delete the token. Keep in mind that this is a public repository. You can create an 'endpoint_local.properties' file to use locally and add it to the '.gitignore'.

Reply:
It is a modified token and the original one is already expired, it is an example of the format in which a token should be, since there was a confusion when generating token on the client side.
It is useful as documentation since the same token will be used in the platform. 
@agreggio-slingr Anyway if it is still inconvenient it can be removed.